python-sdk/src/astro/lineage/facets.py

```diff
@@ -60,7 +60,7 @@ class OutputDatabaseDatasetFacet(BaseFacet):
     columns: list[Column]
     schema: str | None
     used_native_path: bool
-    enabled_native_fallback: bool = False
+    enabled_native_fallback: bool | None = False
```

Comment:
WHY?

Reply:
This can being passed as None from Operator side